src/milind/BankingSystem.java

```diff
@@ -0,0 +1,91 @@
+package milind;
+/*Assignment 09: 
+Create a Banking System which has the following functionality.
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit 
+
+& printBalance method called .
+Hint : Create 3 non static instance variable.
+Output :
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit 
+
+& printBalance method get called.
+Hint : Create 3 static variables will be required.
+Output :
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time*/
+
+public class BankingSystem {
+	String name;
+	int totalBalance;
+	int debit=0;
+	static int totalDebitCount=0;
+	int credit=0;
+	static int totalCreditCount=0;
+	int printBalance=0;
+	static int totalprintBalance=0;
+	
+	void setdata(int intialBal,String cusName)
+	{
+		totalBalance=intialBal;
+		name=cusName;
+	}
+	
+	void debitAmount(int amount) {
+		if(amount<=totalBalance)
+		{
+			totalBalance=totalBalance-amount;
+			debit++;
+			totalDebitCount++;
+		}
+		
+	}
+	
+	void creditAmount(int amount) {
+		if(amount<=totalBalance)
```

Comment:
why this condition?
Is this possible scenario like, if you are credits less amt then total amt then only you can x'fer amt to account?

Reply:
Think on this,